.github/workflows/cloudflare-preview.yml

```
@@ -0,0 +1,36 @@
+on:
+  pull_request:
+    branches:
+      - develop
+name: Cloudflare-preview
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      deployments: write
+      pull-requests: write
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Quarto
+        uses: quarto-dev/quarto-actions/setup@v2
+
+      - name: Render and Publish
+        uses: quarto-dev/quarto-actions/render@v2
+        with:
+          to: html
+      - name: Publish to Cloudflare Pages
```

Is there a reason there are spaces in between previous steps but not here?

```suggestion
          to: html

      - name: Publish to Cloudflare Pages
```

---

Have updated this to have spaces consistent across the steps for this file.